Assignment 3.Rmd

```diff
@@ -88,14 +90,15 @@ Now we need to visualize the clusters we have created. To do so we want to play
 First lets use tidyr to convert from wide to long format.
 ```{r}
 
-K5 <- 
+K5 <- gather(K4,'week','motivation',1:7)
 ```
 
 Now lets use dplyr to average our motivation values by week and by cluster.
 
 ```{r}
 
-K6 <- 
+K6 <- K5 %>% group_by(week,cluster)
+K6 <- summarise(K6,avg=mean(motivation))
```

Comment:
You could pipe these two commands together also if you want, EG:
`K6 <- K5 %>% group_by(week,cluster) %>% summarise(K6,avg=mean(motivation))`